zentral/contrib/osquery/api_views.py

```diff
@@ -416,6 +416,27 @@ def post(self, request, *args, **kwargs):
                         status=status.HTTP_201_CREATED)
 
 
+# Pack Queries
+
+class PackQueryFilter(filters.FilterSet):
+    pack_id = filters.ModelChoiceFilter(field_name="pack_id", queryset=Pack.objects.all())
+    #pack_id = filters.NumberFilter(field_name="pack__id")
```

Comment:
Do not leave a comment here. Also your linter should catch this since a comment must start with `# ` and not `#`.

Reply:
oh, that shouldn't be there